### DISCUSSION TITLE
How to remove source returns because I pay very dearly for searches with the API

### DISCUSSION THREAD

**Discussion by Azoulexxx** [mayooear/ai-pdf-chatbot-langchain#404] [General]
How to remove source returns because I pay very dearly for searches with the API ?

**Comment by Aincvy** [comment]
Modify `makechain.ts`,  `vectorstore.asRetriever()`  =>   `vectorstore.asRetriever(3)`.  3 or 2 or 1.   I believe the result should be based on these sources.
